### PR TITLE
Fixed #32755 -- Corrected Model.get_absolute_url() example in docs.

### PR DESCRIPTION
**Pull request by girishsontakke**
Ticket - [#32755](https://code.djangoproject.com/ticket/32755).

### CODE REVIEW

File: docs/ref/models/instances.txt [hunk 1]
```diff
@@ -714,8 +714,9 @@ usually the best approach.
 For example::
 
     def get_absolute_url(self):
+        # here we assume that you have detail-view as URL pattern name
         from django.urls import reverse
-        return reverse('people.views.details', args=[str(self.id)])
+        return reverse('detail-view', kwargs={"pk" : self.pk})
```
Comment (felixxm) [inline docs/ref/models/instances.txt:719]: I would use `people-detail`, also we shouldn't mix single and double quotes:
```suggestion
        return reverse('people-detail', kwargs={'pk' : self.pk})
```

File: docs/ref/models/instances.txt [hunk 2]
```diff
@@ -714,8 +714,9 @@ usually the best approach.
 For example::
 
     def get_absolute_url(self):
+        # here we assume that you have detail-view as URL pattern name
```
Comment (felixxm) [inline docs/ref/models/instances.txt:717]: IMO we can chop this comment.

**Review by felixxm — COMMENTED** [review]
@girishsontakke Thanks for this patch :+1:

**Comment by girishsontakke** [comment]
@felixxm Thanks for the feedback. I refactor the code.

**Pull request merged**